Example/Tests/MultipartUploadTests.swift

```diff
@@ -41,7 +41,7 @@ class MultipartUploadTests: XCTestCase {
                 let logo = files["logo"] as? String,
                 let jsonString = form["json"] as? String,
                 let data = jsonString.data(using: .utf8),
-                let dict = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: String] {
+                let dict = ((try? JSONSerialization.jsonObject(with: data, options: []) as? [String: String]) as [String : String]??) {
```

Comment:
This doesn't look right

Reply:
This was suggested by XCode Syntax Converter. Does not look necessary. I have made the change.

Reply:
This was suggested by XCode Syntax Converter. Does not look necessary. I have made the change.